tts-dataset-recording-at-home.md

```diff
@@ -0,0 +1,113 @@
+# Guidelines to Record a TTS Dataset at Home
+
+This section describes best practices for getting the best audio quality when recording TTS data at home.
+
+## What data?
+Start by recording the [harvard sentences](http://www.cs.cmu.edu/afs/cs.cmu.edu/project/fgdata/OldFiles/Recorder.app/utterances/Type1/harvsents.txt), which should produce about 20 to 40 minutes of usable data for TTS.
+
+## What gear?
+
+* Use a decent cheap microphone like the [Audio-Technica AT2020USB+](https://www.amazon.com/gp/product/B00B5ZX9FM/ref=ppx_yo_dt_b_asin_title_o00_s00?ie=UTF8&psc=1) or the [blue yeti usb mic](https://www.amazon.com/Blue-Yeti-USB-Microphone-Silver/dp/B002VA464S).
+
+* Get a boom filter or windscreen.
+
+## What audio recording software?
+
+This document focuses on [Audacity](https://www.audacityteam.org/): it’s free, and it has a db meter and shortcuts for making recording easy.
+
+[Reaper](https://www.reaper.fm/) is also quite good but not yet covered on this document.
```

Comment:
please delete if we don't recommend

Reply:
we do recommend this (highly) but don't yet have a full tutorial for it